src/run.js

```diff
@@ -36,13 +37,14 @@ const minimalcss = async options => {
       if (/data:image\//.test(request.url)) {
         // don't need to download those
         request.abort()
-      } else if (/\.(png|jpg|jpeg$)/.test(request.url)) {
+      } else if (/\.(png|jpg|jpeg|webp$)/.test(request.url)) {
         request.abort()
       } else if (stylesheetAstObjects[request.url]) {
         // no point downloading this again
         request.abort()
+      } else if (isInBlacklist(request.url)) {
```

Comment:
This should be...
```javascript
} else if (isInBlacklist(options.blacklist, request.url)) {
```
That way, the "user/client" can control the blacklist. We can certainly maintain a little default list of popular URLs like `www.google-analytics.com`. But if we do maintain a list it should be lifted out and put somewhere else. like in this `src/run.js` file (near the top and call it `BLACKLIST` not `blacklist`)